src/ralph/data_importer/widgets.py

```diff
@@ -217,3 +217,29 @@ def render(self, value):
             value.service.name,
             value.environment.name
         )
+
+
+class RackWidget(widgets.ForeignKeyWidget):
+    """Widget for Rack Foreign Key field.
+
+    CSV field format Rack.name
+    """
+    def clean(self, value):
+        if not value:
+            return None
+        else:
+            try:
+                racks = Rack.objects.all()
+                data = ({'id': rack.id, '__str__': str(rack)} for rack in racks)
+                for datum in data:
+                    if datum['__str__'] == value:
+                        value = Rack.objects.get(id=datum['id'])
+                        break
+            except (ValueError, Rack.DoesNotExist):
+                value = None
+            return value
+
+    def render(self, value):
```

Comment:
Render method must be return ID for export from GUI. Please test this
1. Import/export from GUI
2. Import/export from CLI